PRIVACY_POLICY.md

```diff
@@ -0,0 +1,38 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# Privacy Policy
+
+The Gamepad Navigator allows the user to navigate Chromium-based browsers and web pages using a game controller. It
+provides various features that allow navigation using gamepad such as web page scrolling, tab navigation, click on
+buttons and hyperlinks, et cetera. Due to the nature of browser and web page navigation, the Gamepad Navigator requires
+permission to interact with the content in the active web pages, browser windows, and tabs.
+
+## Information the Gamepad Navigator collects
+
+The Gamepad Navigator provides some web page navigation features that involve interaction with the web page's content.
+To perform certain navigation features (for example, tab navigation) correctly, the elements on the web page may be
+parsed. In addition, the Gamepad Navigator may use the custom gamepad configuration map provided by the user using the
+configuration panel.
+
+## How the Gamepad Navigator uses the information
+
+The Gamepad Navigator is directed by the user's game controller to perform browser and web page navigation according to
+the configuration map set by default or as provided by the user. Some of the navigation features require access to the
+elements on the web page and perform relevant operations, as needed, For example, "tab navigation" requires filtering
+and sorting the tabbable/clickable elements on the web page prior to performing the action.
+
+## What information does the Gamepad Navigator share
+
+The Gamepad Navigator does not explicitly share any information. The gamepad configuration map provided using the
```

Comment:
Why "explicitly"?  You don't share information at all, right?  I would also spell out "information" even if you end up repeating yourself a little.

Reply:
Yes, the extension does not share any information. I just added "explicitly" so that there's no room for confusion for the reader. However, I can re-word the statement if you want.

Reply:
How about this?

> User preferences entered are stored locally on the user's computer and not shared with any site.  Page content used for navigation is only retained temporarily in memory, and is not shared with any site.